### ISSUE TITLE
Add attendees preference for being included in pictures

### ISSUE THREAD

**Issue by 11ma** [codebar/planner#1844]
## Description of the issue 📄
Currently, we ask attendees if they do not want to be included in pictures, but if this preference is already known from their profiles this can help us as organizers be informed of who to exclude from pictures. 
## To do 📋

* [ ] Claim this issue (comment below, or assign yourself if you are part of the codebar org)
* [ ] Fork and clone the repository
* [ ] Update the relevant files. Follow the steps to fix section in this issue.
* [ ] Commit your changes as one commit. Use the title of this issue as your commit message
* [ ] Submit a pull request
* [ ] Mention this issue in the PR description by including it's number
* [ ] Have your pull request reviewed & merged by a codebar team member

**Comment by HelenEBates** [comment]
Hi, I would like to work on this issue, I'm a rails junior dev looking to broaden my experience by working on other projects.

**Comment by KimberleyCook** [comment]
Hey @HelenEBates that would be wonderful please. We would like to ask members at the point when they sign up for codebar and create their account. And then the answer added to their profile somewhere.

Is that ok?

**Comment by HelenEBates** [comment]
@KimberleyCook yes that sounds good, I'm thinking run a migration to add the preference to the members model, and add validation make it required?  I've build the docker image but still need to get the server running, I can see that both the containers are running but when I run `docker logs planner-web-1` there's no output and when I run bin/dstart it doesn't run a server. Does that sound like anything obvious?

**Comment by gnclmorais** [comment]
Hey @HelenEBates, did you end up wrapping this up? Do you have something we can review? :) Cheers!

**Comment by HelenEBates** [comment]
Hi,

I didn't finish it, I can push wjat I have but can't remember how close it
was to done, probably not that close
